epinorm/norm.py

```diff
@@ -282,24 +386,65 @@ def _get_location_from_strain(self, record):
                 return token
 
     def _format_location(self, record):
+        """
+        basically we combine the information from original_record_location_description and the extracted information
+        of the strain. We also clean them (removing unwanted characters)
+
+        we return a dictionaly in json format with all the fields
+        {
+            country
+            areas
+        }
+        """
+
         location = json.loads(record["original_record_location_description"])
-        extracted_location = record["extracted_location"]
-        if "area" not in location:
-            location["area"] = extracted_location
-        country = str(coalesce(location["country"], ""))
-        area = str(coalesce(location["area"], ""))
-        places = []
-        for place in area.split(","):
+
+
+        # format country to make comparison easier
+        location["country"] = location["country"].strip().lower()
+
+        newPlaces = [] # combine areas and extrated location together in this list
```

Comment:
Use `snake_case` :snake: 

```suggestion
        new_places = [] # combine areas and extracted location together in this list
```